.drone.yml

```diff
@@ -16,19 +16,24 @@ trigger:
 
 steps:
 - name: build
-  image: grafana/grafana-plugin-ci:1.8.1-alpine
+  image: grafana/grafana-plugin-ci:1.9.5
   commands:
   - mage -v build
 
 - name: lint
-  image: grafana/grafana-plugin-ci:1.8.1-alpine
+  image: grafana/grafana-plugin-ci:1.9.5
   commands:
   - mage -v lint
 
 - name: test
-  image: grafana/grafana-plugin-ci:1.8.1-alpine
+  image: grafana/grafana-plugin-ci:1.9.5
   commands:
+  - apt update
+  - apt install -y gcc
```

Comment:
`go test -race` requires CGO and a C compiler installed:

```
root@df30cca174ec:/src# mage -v testRace
Running target: TestRace
exec: go "test" "-race" "./..."
go: -race requires cgo; enable cgo by setting CGO_ENABLED=1
```

```
root@df30cca174ec:/src# CGO_ENABLED=1 mage -v testRace
Running target: TestRace
exec: go "test" "-race" "./..."
go: downloading github.com/mitchellh/reflectwalk v1.0.2
# runtime/cgo
cgo: C compiler "gcc" not found: exec: "gcc": executable file not found in $PATH
```

Reply:
Long term I say we should probably stop using those images and use the Go image + mage, and golangci-images, similar to what we do for example in the plugin runner:

https://github.com/grafana/grafana-plugin-runner/blob/main/.drone.yml

Reply:
Weird that it requires CGO. Was this in CI or for you locally?

> Long term I say we should probably stop using those images and use the Go image + mage, and golangci-images

Yeah sounds like a good idea :+1:

Reply:
> Weird that it requires CGO. Was this in CI or for you locally?

Both locally and CI. Unfortunately it's a requirement: https://go.dev/doc/articles/race_detector#Requirements

I wasn't aware of this either before today :sweat_smile:

Reply:
I see. Hopefully things evolve eventually https://github.com/golang/go/issues/9918#issuecomment-1430693077